cypress/integration/e2e/theme-switcher/index.spec.ts

```diff
@@ -1,6 +1,6 @@
 describe('Theme switcher', () => {
   beforeEach(() => {
-    cy.visit('/');
+    cy.visit('/', {timeout: 10000});
```

Comment:
I don't think we should be waiting 10 seconds before each test. It's also only the sidebar test that is flakey, right? The others were fine without the timeout as far as I know. Do we need these anywhere else?

Reply:
I may have misunderstood the `timeout` argument here. I thought it was the time to wait for `cy.visit('/')` to resolve. Not an actual wait 10 seconds. 

I thought the localhost running in the action might be running slow, and not actually resolving in time for the test. Maybe thats not the case. I'll try and bulk these and remove the timeout.